src/wwwroot/app/index.html

```diff
@@ -74,7 +74,7 @@
                      <a ui-sref="candidate"><img src="/app/images/add-user.png">{{::'COMMON.ADD_CANDIDATE' | translate}}</a>
                   </li>
                   <li>
-                     <a ui-sref="vacancy"><img src="/app/images/add.png">{{::'COMMON.ADD_VACANCY' | translate}}</a>
+                     <a ui-sref="vacancyEdit"><img src="/app/images/add.png">{{::'COMMON.ADD_VACANCY' | translate}}</a>
                   </li>
```

Comment:
This problem in index.html will be fixed in next pull request - "Feature/menu directive"